### ISSUE TITLE
Admin UI

### ISSUE THREAD

**Issue by matrixik** [AntoineAugusti/feature-flags#8]
Hi, nice tool. Thank you for working on it.
Would be nice to have some admin UI.

There is Unleash that you could look into https://github.com/Unleash/unleash
It's using https://github.com/Unleash/unleash-client-go for Go client.

**Comment by AntoineAugusti** [comment]
Definitely, it'd be super convenient and it's totally missing. At the moment I don't have time to work on it unfortunately 😞
